install/bootstrap-garage.sh

```diff
@@ -0,0 +1,25 @@
+echo "${_group}Bootstrapping garage (node store)..."
+
+$dc up --wait garage
+garage="$dc exec garage /garage"
+
+if [[ $($garage bucket list | tail -1 | awk '{print $1}') != 'nodestore' ]]; then
+  node_id=$($garage status | tail -1 | awk '{print $1}')
+  $garage layout assign -z dc1 -c 100G "$node_id"
```

Comment:
Should this `100G` be a variable somewhere?

Reply:
Yes, I think we should add a new `GARAGE_STORAGE_SIZE` env var to `.env`. That said not sure if that makes much sense as we would not honor any changes to that after the initial installation. Unless this actually _reserves_ 100G, I think leaving it hard-coded to a "good enough" value and then documenting how to change this if needed would be a better option.

Thoughts?